runtime/testing-hydradx/src/lib.rs

```diff
@@ -447,23 +745,50 @@ construct_runtime!(
 		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 2,
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 3,
 		Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>} = 4,
-		Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 5,
 		Utility: pallet_utility::{Pallet, Call, Event} = 6,
+		Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 7,
```

Comment:
keep this 5

Reply:
give pallets some buffer, use only odd numbers and add 20 between sections

Reply:
What's the reason behind using only odd numbers? I'm just curious.

Reply:
we can always sneak in one pallet if necessary as some pallets require specific order

Reply:
done